src/ddoc_cache/src/ddoc_cache_entry_custom.erl

```diff
@@ -10,25 +10,28 @@
 % License for the specific language governing permissions and limitations under
 % the License.
 
--module(ddoc_cache_util).
+-module(ddoc_cache_entry_custom).
```

Comment:
Not sure on the dispatch question. The diff there is a bit unfortunate as I removed an unused util module and added this one that's smaller. The custom module is part of the API though. Mango uses this to cache all indexes for a database.